node/builder_chain.go

```diff
@@ -361,6 +355,18 @@ func ConfigFullNode(c interface{}) Option {
 				Override(InitChainIndexerKey, modules.InitChainIndexer(cfg.ChainIndexer)),
 			),
 		),
+
+		If(cfg.PaymentChannels.EnablePaymentChannelManager,
+			Override(new(paychmgr.ManagerNodeAPI), From(new(modules.PaychManagerNodeAPI))),
```

Comment:
I'm seeing this is a bit different than the removal up above.  Is this part of an internal refactor?

Reply:
Looks like paychmgr.ManagerNodeAPI added

Reply:
and API instantiation comes thorugh PaychImpl now

Reply:
Ok nice explained here: https://github.com/filecoin-project/lotus/pull/13139#issuecomment-3003200684